reg/drone/service/actuator/servo/_.0.1.uavcan

```diff
@@ -0,0 +1,97 @@
+# A servo can actuate either a translational or rotary load using electric power from the high-voltage DC bus.
+#
+# The type of load (translational or rotary) dictates which type is used for commanding the setpoint and reporting
+# the status:
+#   - reg.drone.physics.kinetics.Linear1DTs
+#   - reg.drone.physics.kinetics.Angular1DTs
+# For generality, either or both of these types are referred to as "*1DTs" (timestamped) or "*1D" (non-timestamped).
+#
+# The subjects defined by this service are shown on the following canvas. Implementers are encouraged to add
+# custom subjects with additional data. Notice that the physics subjects are timestamped.
+#
+#                    SUBJECT                            TYPE                                            RATE
+#
+#   +------------+   setpoint       +------------+      reg.drone.physics.kinetics.*1D                  R
```

Comment:
Should this be:
```suggestion
#   +------------+   setpoint       +------------+      reg.drone.service.actuator.common.sp.*          R
```

Reply:
No, this is not a paste error, this is intentional. As described a bit further down in the same file, reliance on the same type allows the controller to set the motion profile of the servo, which is commonly used in industrial applications. Here is a related excerpt from an industrial CANopen servo documentation:

<img width="784" alt="image" src="https://user-images.githubusercontent.com/3298404/95511075-9f3e5d00-09bf-11eb-901b-f040abaebb74.png">

If there is interest, we can also define group control modes for servos similar to what you suggested (without the motion profile setting capability), but I am not sure what the common requirements are.

Reply:
@pavel-kirienko previously in DS-015 we have defined group commands similar to the ESCs. Can we please add the same here as well?

Reply:
Done.